llvm/lib/Target/AArch64/AArch64ISelDAGToDAG.cpp

```diff
@@ -1859,6 +1867,34 @@ void AArch64DAGToDAGISel::SelectFrintFromVT(SDNode *N, unsigned NumVecs,
   SelectUnaryMultiIntrinsic(N, NumVecs, true, Opcode);
 }
 
+template <int64_t Max>
```

Comment:
Rather than create two almost identical copies of the function with a template parameter, I think in this case it makes sense to just pass Max in as a function argument.

Reply:
Done